Documentation/5.4.1/README.rst

```diff
@@ -703,7 +704,49 @@ In OSPD 13 and later, /usr/share/openstack-tripleo-heat-templates/environments/n
         ...
 
 
-7. Please follow **Phase 6** steps again for verfication of all the nodes are assigned with correct flavors.
+7. **(Optional)** To enable Smart NIC Integration, perform the following instrctions:
+
+:Step 1: Create a new sriov-role.yaml file to deploy SR-IOV Compute nodes. The command used to create this file is:
```

Comment:
Should these all be called ovrs-role.yaml, ComputeOvrs, etc. rather than sriov?  Or is this so close to an SRIOV flavor that it should be named the same?

Reply:
VRS offload requires the same serivces that are available in ComputeSriov role. That is why we didn't got a chance to create a new role ComputeOvrs.
Also, Mellanox documentation is suggesting users to use ComputeSriov role for OVS offload.
http://www.mellanox.com/related-docs/prod_software/Mellanox_Support_for_TripleO_Queens_Application_Notes_v1.0.pdf

Reply:
OK, makes sense.